### PR TITLE
#593 Annoying SyntaxWarning

### PR DESCRIPTION
**Pull request by v01d-gh**
#593

### CODE REVIEW

**Comment by v01d-gh** [comment]
#593

**Review by junkmd — COMMENTED** [review]
Thank you for your contribution.

Currently, the changelog for this project is updated by the maintainer right before the release of a new version.
I will revert the change that bumped the version to 1.4.6.

When a new version is released, I will certainly include your contribution in the changelog.

File: CHANGES.txt [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Comtypes CHANGELOG
 ==================
 
+Release 1.4.6
+--------------
+* Fix Annoying SyntaxWarning. By @v01d-gh.
+
 Release 1.4.5
```
Comment (junkmd) [inline CHANGES.txt:8]:
```suggestion
Release 1.4.5
```

File: comtypes/__init__.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # comtypes version numbers follow semver (http://semver.org/) and PEP 440
-__version__ = "1.4.5"
+__version__ = "1.4.6"
```
Comment (junkmd) [inline comtypes/__init__.py:2]:
```suggestion
__version__ = "1.4.5"
```

File: docs/source/conf.py [hunk 3]
```diff
@@ -50,7 +50,7 @@
 # The short X.Y version.
 version = '1.4'
 # The full version, including alpha/beta/rc tags.
-release = '1.4.5'
+release = '1.4.6'
```
Comment (junkmd) [inline docs/source/conf.py:53]:
```suggestion
release = '1.4.5'
```

**Review by junkmd — APPROVED** [review]

**Pull request merged**